### DISCUSSION TITLE
UI-Lovelace-Minimalist Figma

### DISCUSSION THREAD

**Discussion by sildehoop** [UI-Lovelace-Minimalist/UI#83] [Show and tell]
I have made an Figma with almost all the components this way you can make pictures for on the wiki
https://www.figma.com/file/dOQKR8DCP2mvH7zesjU8L6/UI-Lovelace-Minimalist?node-id=0%3A1

**Comment by CM000n** [comment]
That looks really interesting! Unfortunately, Figma doesn't mean anything to me personally 😁. How can you create the cards there? Did you have to build them manually or is there a way to generate them directly from the .yaml configs?

**Reply by sildehoop** [reply]
Do you now Adobe XD ??

**Reply by sildehoop** [reply]
Is is just for web designing so no yaml
